### PR TITLE
docs: Fix typo in core::0141::forbidden-types

### PR DESCRIPTION
**Pull request by apasel422**
Though it's not clear to me whether the suppression comment should be added over the field or message.

### CODE REVIEW

File: docs/rules/0141/forbidden-types.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ message Book {
 
 ## Disabling
 
-If you need to violate this rule, use a leading comment above the method.
+If you need to violate this rule, use a leading comment above the message.
```
Comment (lukesneeringer) [inline docs/rules/0141/forbidden-types.md:52]:
```suggestion
If you need to violate this rule, use a leading comment above the field.
```

**Review by lukesneeringer — APPROVED** [review]
It should be field. Fixed it.

**Comment by apasel422** [comment]
The example currently puts the comment above the message, though.

**Comment by lukesneeringer** [comment]
Oh, thanks.

**Pull request merged**